.gitignore

```
@@ -14,8 +14,8 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
-lib64/
+# lib/
+# lib64/
```

suggestion: if you still want to ignore the top-level versions of these files, you can prefix them with `/`:
```suggestion
/lib/
/lib64/
```